compute/admin_guide/alerts/alert_mechanism.adoc

```diff
@@ -1,98 +1,76 @@
-
-Not all triggers are available for all channels.
-For example, new JIRA issues can only be opened when vulnerability rules are triggered.
-
-=== Prisma Cloud Notification provider integration
-
-You can use the same notification settings set up in the platform for CSPM alerts by importing them into Compute in an Alert Profile. 
+There are two ways of integrating with alert providers. 
 
-These configurations are setup in the platform under **Settings > Integrations** tab and cannot be configured from within Compute. Any changes to the provider settings will need to be done on the platform side.
+. Set up once on the platform under *Settings > Integrations* for all the https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/manage-prisma-cloud-alerts/send-prisma-cloud-alert-notifications-to-third-party-tools[supported integrations] and use the same integration for sending both CSPM and Compute alerts.
 
-==== Importing platform configurations inside Compute
+. Set it up on *Compute > Manage > Alerts > Manage* for integrations that are only available on Compute.
+For example, if you want to use the IBM Cloud Security Advisor, or Cortex as your alert provider.
 
-To learn more about setting up integration / alert providers in the platform, follow steps documented https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/configure-external-integrations-on-prisma-cloud.html#id24911ff9-c9ec-4503-bb3a-6cfce792a70d[here].
+endif::prisma_cloud[]
 
-. Navigate to **Manage > Alerts** tab in Compute, click on "Add Profile".
-. From the *Provider* drop down, select *Prisma Cloud*.
-. In the *Integrations* field, select the configuration you want to choose.
-. Select triggers to be sent to this channel.
+You can create any number of alert profiles, where each profile gives you granular control over who should receive the notifications and for what types of alerts.
 
-And hit Save. 
+Alert settings::
+Specifies the configuration settings required to send the alert to the alert provider or messaging medium.
 
-NOTE: Test alert notifications are sent immediately to the provider channels regardless of alert aggregation period chosen.
-
-==== Supported Prisma Cloud Integrations
 
-* Email
-* JIRA
-* Slack
-* Splunk
-* PagerDuty
-* Webhooks
-* Google Cloud Security Command Center - Only available for https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/connect-your-cloud-platform-to-prisma-cloud/cloud-account-onboarding.html[onboarded PC accounts].
-* AWS Security Hub - Only available for onboarded PC accounts.
-* ServiceNow - Only Incident Response
+Alert triggers:: 
+Specifies what alerts you want to send to the provider included in the profile.
+Alerts are generated when the rules included in your policy are violated, and you can choose whether you want to send a summarized notification for the detected issues. For example, on runtime violations, compliance violations, cloud discovery or WAAS.
```

Comment:
@Pubs-MV not sure about "summarized", maybe we can remove it, since triggers are acting diferently